### PR TITLE
Reduce typo count.

### PR DESCRIPTION
**Pull request by waywardmonkeys**
This renames 2 identifiers, one in benches, one in Rayon support. Neither are public.

### CODE REVIEW

File: src/raw/mod.rs [hunk 1]
```diff
@@ -1788,7 +1788,7 @@ impl RawTableInner {
     ///
     /// This function does not check if the given element exists in the table. Also,
     /// this function does not check if there is enough space in the table to insert
-    /// a new element. Caller of the funtion must make ensure that the table has at
+    /// a new element. The caller of the function must make ensure that the table has at
```
Comment (cuviper) [inline src/raw/mod.rs:1791]: This should either be "make sure" or "ensure".
```suggestion
    /// a new element. The caller of the function must make sure that the table has at
```

File: src/raw/mod.rs [hunk 2]
```diff
@@ -3171,7 +3171,7 @@ impl<T: Clone, A: Allocator + Clone> Clone for RawTable<T, A> {
                 // Avoid `Result::ok_or_else` because it bloats LLVM IR.
                 //
                 // SAFETY: This is safe as we are taking the size of an already allocated table
-                // and therefore сapacity overflow cannot occur, `self.table.buckets()` is power
+                // and therefore capacity overflow cannot occur, `self.table.buckets()` is power
```
Comment (cuviper) [inline src/raw/mod.rs:3174]: How strange, that first "c" was a U+0441 Cyrillic Es before!

**Review by cuviper — APPROVED** [review]
LGTM, just one additional nit if you don't mind.

File: src/raw/mod.rs (hunk 2 above)
Comment (waywardmonkeys) [reply src/raw/mod.rs:3174]: Indeed!

**Pull request merged**